proposals/datetime-picker-tools.md

```diff
@@ -0,0 +1,147 @@
+# baseui/datepicker
+
+This is a proposal for enhancements to the existing collection of datepicker components.
+
+## Motivation
+
+Date/time selections need more precision than a single date or range of dates (what is currently implemented). This proposal outlines two new components and changes to one existing component. `TimePicker`, `TimezonePicker`, and `DatePicker`.
+
+## APIs and Components
+
+`TimePicker` - used to select a time, decoupled from a date. E.g. `11:50AM`.
+
+API:
+
+* `precisionInMinutes`: `number` Range from 1 - 60. 15 default.
```

Comment:
`minuteStep` might work better? Other libraries tend to use it and it seems more obvious? 

Also, did we consider including seconds? I think sooner or later somebody will request that. Then, it might make sense to keep `step` in seconds.

Reply:
changed to `step` in seconds. also changed `callback` argument and `value` prop to simply be a number